releasenotes/notes/wasm-support-gcr-ecr-acr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: extensibility
+issue: []
+releaseNotes:
+  - |
+    **Added** Support for ImagePullPolicy of WasmPlugin API. 
```

Comment:
This is confusing. We already have this - but you have added native support for AWS/AZure repos right?

Reply:
My bad. The file was not saved and submitted. Done to fix it. Thanks.